### ISSUE TITLE
Seems like 13.0.0-beta.1 not supported

### ISSUE THREAD

**Issue by JeonDohyeon** [shrimpia/misskey-tools#74]
Throws an error "unknown exception" on login. Happens on both new and old users.

**Comment by EbiseLutica** [comment]
メモ：13.0.0-beta.14 でも再現した

**Comment by JeonDohyeon** [comment]
Being fixed on  https://github.com/misskey-dev/misskey/pull/9462, closing when merged

**Comment by JeonDohyeon** [comment]
> Being fixed on [misskey-dev/misskey#9462](https://github.com/misskey-dev/misskey/pull/9462), closing when merged

Pull request reverted, continuing

---

Checked packets, and it attempts to access https://misskey.tools/api/v1/session/ with:
![image](https://user-images.githubusercontent.com/34373595/210665896-ab3ecc68-9923-4129-99a7-bcad453d7577.png)

It returns an error "AuthorzationRequiredError" like:
```json
{"name":"AuthorizationRequiredError","message":"Authorization is required for request on GET /api/v1/session/","stack":"Error\n    at new HttpError (/opt/misskey-tools/node_modules/routing-controllers/cjs/http-error/HttpError.js:17:22)\n    at new UnauthorizedError (/opt/misskey-tools/node_modules/routing-controllers/cjs/http-error/UnauthorizedError.js:10:9)\n    at new AuthorizationRequiredError (/opt/misskey-tools/node_modules/routing-controllers/cjs/error/AuthorizationRequiredError.js:10:9)\n    at /opt/misskey-tools/node_modules/routing-controllers/cjs/ActionParameterHandler.js:71:51\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
```
(always same response)

**Comment by JeonDohyeon** [comment]
Seemingly completed, closing issue
